packages/paste-website/src/pages/tokens/theme-package.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Theme Package
+package: '@twilio-paste/theme'
+description: The Paste theme package allows easy and simple access to the paste design tokens via a theme provider to react components
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/theme/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteThemePackage(filter: {name: {eq: "@twilio-paste/theme"}}) {
+      edges {
+        node {
+          name
+          description
+          version
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Theme package"
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/button"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/overview-theme--provider"
+  data={props.data.allPasteThemePackage.edges}
+/>
+
+***
+
+<content>
+
+## Theme Provider
+
+Paste Components require you to wrap your application in the Paste Theme provider to apply the theme tokens to the components you use, and provide a way to access Paste Design Tokens in your custom components.
+
+The Paste Theme Provider is a wrapper around the [Emotion Theme Provider](https://emotion.sh/docs/emotion-theming) and comes with a default theme.
+
+You should place the `Theme.Provider` around the root of your React application.
+
+```js
+import {Theme} from '@twilio-paste/theme';
+
+<Theme.Provider>
+  ...
+</Theme.Provider>
+```
+
+### Using tokens in custom components
+
+Sometimes you will need to create something custom that is not available in Paste but you need that component to still look like it's from Paste. The use of Paste Tokens in your styles is the way that you can access global design properties used in all Paste Components.
+
+#### Emotion Styled Component
+
+By using the `Theme.Provider`, when you create a custom component using Emotion CSS, the theme object is available on `props` via context. You can access those via the [Styled-System `themeGet` utility](https://styled-system.com/api#themeget) or props object directly.
+
+This is the preferred method.
+
+```js
+import styled from '@emotion/styled';
+import { themeGet } from 'styled-system';
+
+const custom = styled.div`
+  background-color: ${themeGet('backgrounds.primary')};
+  padding: ${props => props.theme.spacing.spacing20};
+`
+```
+
+#### Paste ThemeConsumer
+
+Paste Theme provides a React Context Consumer for the theme called `Theme.Consumer`. The `Theme.Consumer` takes a function as a child, which provides the theme object as an argument.
```

Comment:
Add a line about when would a customer use this as opposed to preferred method.

Reply:
Gunna bundle the "reason" for all three under one umbrella, because it's if you're not using emotion or for whatever reason you want the token value in some logic.

Reply:
Great!